quiz.js

```diff
@@ -10,20 +10,38 @@ var info = {};
 var button = document.getElementById("click");
 button.addEventListener("click", argument);
 
-function argument(clickEvent) {
-	clickEvent.preventDefault();
+function heightAlert () {
+	if (info.height === "" || info.height === " " || info.height === 0) {
+			alert("Both fields must have a value");
+	}
+}
+function characterAlert () {
+	if (info.character === "" || info.character===" ") {
+		alert("Both fields must have a value");
+	}
+}
+
+
+function argument(click) {
+	click.preventDefault();
 	info.height = parseInt(height.value);
```

Comment:
Always use the radix parameter for parseInt -> `parseInt(height.value, 10)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt